CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The exercism CLI follows [semantic versioning](http://semver.org/).
 ----------------
 
 ## Next Release
-* **Your contribution here**
+* [#736](https://github.com/exercism/cli/pull/736) Metadata file .solution.json renamed to metadata.json
```

Comment:
Nice.

Reply:
@jdsutherland we normally add the  contributor name to the changelog entry `- [@jdsutherland]`.